source/main/cardano/nodes/jormungandr.js

```diff
@@ -18,10 +17,11 @@ export function buildJormungandrNodeOpts({ nodePort, stateDir }: WalletOpts) {
     // '--random-port',
     '--port',
     '8088',
-    '--genesis-block',
-    `${stateDir}/block0.bin`,
-    '--bft-leaders',
-    `${stateDir}/secret.yaml`,
+    '--genesis-block-hash', 'adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770', // TODO, daedalus should either get the hash from launcher-config.yaml, or the the entire nodeArgs array
```

Comment:
Using nodeArgs to handle the difference between the selfnode and testnet make sense. We can just append nodeArgs onto the array at this point. These are still available from `CardanoNode`, so its just a matter of passing them through